daq/topology.py

```diff
@@ -42,6 +42,7 @@ class FaucetTopology:
     PRI_TRUNK_NAME = 'trunk_pri'
     _NO_VLAN = "0x0000/0x1000"
     _EXT_STACK = 'EXT_STACK'
+    _IN_PORT = 0xfffffff8
```

Comment:
Maybe we should use the the full name OFPP_IN_PORT here. in_port is a bit confusing on its own.

Reply:
Agreed. Done.